homeassistant/components/homekit_controller/manifest.json

```diff
@@ -3,9 +3,10 @@
   "name": "HomeKit Controller",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/homekit_controller",
-  "requirements": ["aiohomekit==1.1.1"],
+  "requirements": ["aiohomekit==1.1.4"],
   "zeroconf": ["_hap._tcp.local.", "_hap._udp.local."],
-  "after_dependencies": ["zeroconf"],
+  "bluetooth": [{ "manufacturer_id": 76, "manufacturer_data_first_byte": 6 }],
+  "after_dependencies": ["zeroconf", "bluetooth"],
```

Comment:
We can drop Bluetooth here since it’s a conditional load and we don’t want to load it right now since we want users to opt in currently

Reply:
6aa6c3132d5655748dca6ca593dddc7a440dbcae